src/lib/components/Matching.svelte

```
@@ -1,300 +1,322 @@
+						<div class="dropdown flex-1">
+							<button
+								type="button"
+								class="btn select select-bordered w-full rounded-full {qs.showSolution &&
+								isSelectionCorrectForPrompt(p.id)
+									? 'select-success'
+									: ''} {qs.showSolution ? 'btn-disabled' : ''}"
+							>
+								{#if qs.showSolution && correctAnswerIdForPrompt(p.id)}
+									{@html (
+										answers().find((a) => a.id === correctAnswerIdForPrompt(p.id))?.text || ''
+									).slice('answer:'.length)}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)